tests/test_batch.py

```diff
@@ -866,3 +867,47 @@ def test_get_batches_raises_value_error_on_invalid_value_for_limit(
         """
         with pytest.raises(ValueError, match="limit"):
             _ = self.sdk.get_batches(pagination_params=PaginationParams(limit=limit))
+
+    @pytest.mark.parametrize(
+        "batch_ids",
+        [
+            # No batch id provided
+            [],
+            # Single string UUID for id
+            [str(UUID(int=0x1))],
+            # List of string UUIDs for id
+            [str(UUID(int=0x1)), str(UUID(int=0x2))],
+        ],
+    )
+    def test_cancel_batches_success(
+        self,
+        mock_request: Any,
+        batch_ids: Any,
+    ):
+        """
+        As a user using the SDK with proper credentials,
+        I can cancel of a group of batches from a batch with specific batch ids.
+        The resulting request will retrieve the batches that were cancelled and
+        the errors for those that could not be cancelled.
+        """
+        response = self.sdk.cancel_batches(batch_ids=batch_ids)
+        assert isinstance(response, BatchCancellationResponse)
+
+        for item in response.batches:
+            assert isinstance(item, Batch)
+
+        for k, v in response.errors.items():
+            assert isinstance(k, UUID)
+            assert isinstance(v, str)
```

Comment:
If I'm not mistaken in your response fixture you're not including any errors so this is never really tested.